src/plugins/dashboard/public/application/legacy_app.ts

```diff
@@ -47,15 +50,34 @@ import { DashboardListing, EMPTY_FILTER } from './listing/dashboard_listing';
 import { addHelpMenuToAppChrome } from './help_menu/help_menu_util';
 import { syncQueryStateWithUrl } from '../../../data/public';
 
+export interface DashboardListItem {
```

Comment:
Why are we declaring the same interface twice? you have already declared it in [src/plugins/dashboard/common/types.ts](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3090/files#diff-dc3f3df85647adf7f9b405affdac458487c272a38e496c4df9fa2bd86715b15d)

Reply:
My mistake.  I moved the type decl.. missed the cleanup.